### DISCUSSION TITLE
[Esame di prova 18 dicembre 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#424] [Esami M-Z]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/071a5d6c-9424-4f0f-ac3b-8d4f19b40e49)

**Comment by Frisbon** [comment]
![image_2024-01-26_15-24-02](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/aa912425-72be-49cf-a8dd-12c966364bfa)

Per cui tutte le soluzioni sono date da $S = \{ 140 + 299k\  , k \in \mathbb{Z} \}$

Proof:
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/ef608d6a-2b84-429b-b5ad-d02526617839)
